### ISSUE TITLE
Opendeck 1.2.0 stuck in bootloader

### ISSUE THREAD

**Issue by Solapse** [shanteacontrols/OpenDeck#134]
Hi, 

I was updating the Opendeck via the mass storage to the last supported 4.0 release and it is stuck in bootloader mode. Any tips on how to resolve this short of doing a manual method via arduino?

If not - I couldn't see any documentation for this board any longer - are there any resources to explain what the pinouts are (I'm guessing it is CONN2)?

**Comment by Solapse** [comment]
Thanks. I've had a go at flashing it, but I'm having problems with AVRdude. It complains that avrconf: "programmer type linuxspi not found". I've looked around and can see that the reset pin is set to 25 in the config.

I'm using a Mega 2560 to flash - followed the guide and added the capacitor as instructed, and put the reset pin to pin 10.

I vaguely remember had some problems like this a long time ago but can't recall what I did to resolve it,
